generators/bdk/java/templates/ext-app/application.yaml.ejs

```diff
@@ -0,0 +1,49 @@
+# BDK Core Configuration
+bdk:
+  host: <%= host %>
+  bot:
+    username: <%= username %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+  app:
+    appId: <%= appId %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+# BDK App Configuration
+bdk-app:
+  auth:
+    enabled: true
+    jwtCookie:
+      enabled: true
+      maxAge: 1d
+  cors:
+    "[/**]":
+      allowed-origins: "*"
+      allowed-credentials: false
+  tracing:
+    enabled: true
+    urlPatterns:
+      - /api/*
+
+logging:
+  pattern:
+    # Added %clr(%X{X-Trace-Id}){magenta} to output the MDC traceId in logs
+    console: '%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr(${PID}){magenta} %clr(%X{X-Trace-Id}){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx'
+  level:
+    com.symphony: debug
+
+server:
+  port: 10443
+  ssl:
+    key-store: classpath:keystore.p12
+    key-store-password: password
+    key-store-type: pkcs12
+    key-alias: tomcat
+
+spring:
+  security:
+    user:
+      name: admin
+      password: password
```

Comment:
what is it used for? do we need it?

Reply:
I followed the app spring starter example in the bdk repo 😅 I'm not sure about this config TBH